components/eam/src/chemistry/modal_aero/seasalt_model.F90

```diff
@@ -469,7 +408,6 @@ subroutine seasalt_emis(u10, u10cubed, lchnk, srf_temp, ocnfrc, ncol, cflx, emis
 
     fi(:ncol,:nsections) = fluxes( srf_temp, u10cubed, ncol )
 
-    calculate_organic_fraction: if ( has_mam_mom ) then
 
        nullify(chla)
```

Comment:
please change the indent as the if condition is removed. same as other places.

Reply:
changed the indent for subroutines.